src/test/java/teammates/test/cases/action/InstructorFeedbackQuestionEditActionTest.java

```diff
@@ -971,6 +963,122 @@ public void testExecuteAndPostProcessConstSumOption() {
+
+        ______TS("edit min points larger than max points constraint -> edit failure ");
+
+        additionalParams = new String[] {
+                Const.ParamsNames.FEEDBACK_QUESTION_TEXT, "Split points among the options.(edited)",
+                Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTS, "1000",
+                Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTSFOREACHOPTION, "300",
+                Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTSFOREACHRECIPIENT, "500",
+                Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTS_HAS_MINPOINTS_CONSTRAINT, "on",
+                Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTS_HAS_MAXPOINTS_CONSTRAINT, "on",
+                Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTSMIN, "30",
+                Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTSMAX, "29"
+        };
+
+        params = ArrayUtils.addAll(baseParams, additionalParams);
+        a = getAction(params);
+        r = getRedirectResult(a);
+        assertTrue(r.isError);
+
+        assertEquals(
+                getPageResultDestination(
+                        Const.ActionURIs.INSTRUCTOR_FEEDBACK_EDIT_PAGE,
+                        "FSQTT.idOfTypicalCourse1",
+                        "CONSTSUM+Session",
+                        "FSQTT.idOfInstructor1OfCourse1",
+                        true),
+                r.getDestinationWithParams());
+        assertEquals(Const.FeedbackQuestion.CONST_SUM_ERROR_MIN_CONSTRAINT_GREATER_THAN_MAX, r.getStatusMessage());
+
+        ______TS("edit min points less than 0 -> edit failure ");
+
+        additionalParams = new String[] {
+                Const.ParamsNames.FEEDBACK_QUESTION_TEXT, "Split points among the options.(edited)",
+                Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTS, "1000",
+                Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTSFOREACHOPTION, "300",
+                Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTSFOREACHRECIPIENT, "500",
+                Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTS_HAS_MINPOINTS_CONSTRAINT, "on",
+                Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTSMIN, "-1",
+        };
+
+        params = ArrayUtils.addAll(baseParams, additionalParams);
+        a = getAction(params);
+        r = getRedirectResult(a);
+        assertTrue(r.isError);
+
+        assertEquals(
+                getPageResultDestination(
+                        Const.ActionURIs.INSTRUCTOR_FEEDBACK_EDIT_PAGE,
+                        "FSQTT.idOfTypicalCourse1",
+                        "CONSTSUM+Session",
+                        "FSQTT.idOfInstructor1OfCourse1",
+                        true),
+                r.getDestinationWithParams());
+        assertEquals(String.format(Const.FeedbackQuestion.CONST_SUM_ERROR_POINTS_CONSTRAINT_LESS_THAN_ZERO, "Min"),
+                r.getStatusMessage());
+        // All existing responses should be deleted as the options are edited
```

Comment:
Irrelevant comment? // All existing responses should be deleted as the options are edited
You should validate the question is not being edit right?